package.json

```
@@ -14,6 +14,9 @@
     "test": "jest",
     "test:routes": "jest -c jest.routes.config.js",
     "test:a11y": "jest -c jest.a11y.config.js --maxWorkers 15",
+    "sonar-scan": "sonar-scanner",
+    "test:smoke": "exit 0",
```

this should really have the appropriate jest config here for tests
but it shouldn't silently pass the build
```suggestion
    "test:smoke": "echo 'TODO add smoke tests' && exit 1",
```